tests/EPPs/test_make_kapa_csv.py

```diff
@@ -79,32 +57,20 @@ def test_get_file_data_and_write_Enzymatic_fragmentation(
 
 
 def test_get_file_data_and_write_missing_udf(
-    lims, kapa_library_preparation_file_missing_udf, helpers, kapa_csv_data
-):
-    # GIVEN: A file name. and a lims with three samples that has been run
-    # through a process of type <amount_step>. There some samples did not get the udf: Amount needed (ng) set.
-    # The tree samples are now in another step where they have location and
+        server_make_kapa_csv_missing_udfs, kapa_library_preparation_file_missing_udf, lims):
```

Comment:
Adding type hints makes also the tests much easier to read!
I think my type hints suggestions are wrong but just exemplifying 

```suggestion
        server_make_kapa_csv_missing_udfs: str, kapa_library_preparation_file_missing_udf: str, lims: Lims):
```